TAGS

```diff
@@ -18,3 +18,5 @@
 6.6/slim:6.6.6-slim:6.6-slim 6-slim
 5.5:5.5.5:5.5 5
 5.5/slim:5.5.5-slim:5.5-slim 5-slim
+8.7:8.7.0:8.7 8 latest
+8.7/slim:8.7.0-slim:8.7-slim 8-slim latest-slim
```

Comment:
These should be sorted at the top; the file appears to be in descending version order. Did the scripts not do that?

Reply:
I simply ran the scripts, following update.md instructions.

Reply:
I moved it manually.  Can you approve please?